cerner/terra-core
# mouseDown touch-input NaN/Null Fix

### Summary
 
The reason for these changes is to fix an issue with signing on windows touch screen devices. On mouseDown was not done correctly which leads to Nan Values. 

**A description of the original issue:**
esignature on the web would not allow for signing via touch on any windows based device with touch screen capabbilities (i.e surface pro). Fixed a bug an a preexisting pr. 

**Steps to reproduce the original issue:**
1. Boot up the terra dev site.
2. go into signature. 
3. add console log for the lineSegements on mouse down.
4. attempt to make two seperate lines with touch inputs and then check the values of the line segments. it contains Nan's. 

**A description of the changes that have been made, why they were made, and how they are resolving the issue:**
2. Added checks for touchInputs vs mouseInputs on mouseDown. 

**Steps to validate the changes are working as expected:**
1. Boot up the terra dev site.
2. go into signature. 
3. add console log for the lineSegements on mouse down.
4. attempt to make two seperate lines with touch inputs and then check the values of the line segments. it should no longer contain nan's. 

### Testing

This was done manually on:
Ipad, Surface pro, and Mac using safari, chrome, IE, and Edge

### Additional Details

A fix on a previous Pr -> https://github.com/cerner/terra-core/pull/3391 



Thank you for contributing to Terra.
@cerner/terra
